### DISCUSSION TITLE
Adding other webservers running on other ports to my EE domain using subdomains

### DISCUSSION THREAD

**Discussion by woobenjamin** [EasyEngine/easyengine#1661] [Support Requests]
Hi All,

Thank you for developing EasyEngine. I love it! I am now running multiple WordPress sites on one server without multisite!! 

I'm pretty sure I'm not the first to ask this, but I can't seem to find an answer despite all my Googling! Anyway, this is what I'm trying to achieve ...

Let's say I have 3 WordPress sites already set up using EE with LE SSL - let's call them site1.com, site2.com, site3.com.
Let's say I want to run 3 webservers that listen on ports 5555, 6666, 7777 respectively on site2.com.
  - for these webservers, I want to get to them using a subdomain (i.e. webserver1.site2.com, webserver2.site2.com, webserver3.site2.com)
  - I'm hoping that NGINX ensures that the connection between the client and my server is secure, so that I can essentially access these servers using http**s**://webserver1.site2.com, http**s**:;//webserver2.site2.com, and http**s**://webserver3.site2.com.

This is where I am lost. 
Where and what do I have to add or edit? 

I _think_ it will be /opt/easyengine/sites/site2.com/config/nginx/custom.conf. 

Finally, my webservers are run in a Docker stack outside of EE. So, there is a possibility that if someone typed http(s)://IP_ADDRESS:5555 or 6666 or 7777 they might reach them (since my Portainer app I can reach by just using IP_ADDRESS:9000, but one of my other webservers doesn't seem to work because it is not on a popular port).

If someone could help point me in the right direction, I'm happy to keep Googling and learning more about what I need to do.

Many thanks in advance!

b

**Comment by jauhari** [comment]
We need this one too...

**Comment by jauhari** [comment]
any update about it?
